packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="JetBrains.Annotations" version="8.0.5.0" targetFramework="net20" />
```

Comment:
I think you got rid of the Nuget package incorrectly.  If you opened references and deleted the dll reference, that does not remove it from Nuget.  You have to do that from the Manage Nuget dependencies dialog.  This is an artifact that says your WinSW project still requires, via Nuget JetBrains.Annotations